### PR TITLE
[16.0][MIG] crm_security_group: Migration to 16.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 16.0

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT45865

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
Tested in http://oca-crm-16-0-pr526-f7ca87ef1bda.runboat.odoo-community.org with a specially created user crm/crm

File: crm_security_group/readme/ROADMAP.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+- Remove sale_crm dependency (now is not possible), maybe split these part in other addon
```
Comment (chienandalu) [inline crm_security_group/readme/ROADMAP.rst]: nitpick:
```suggestion
- Remove sale_crm dependency (now is not possible), maybe split this part in other addon
```

**Review by chienandalu — APPROVED** [review]
Working fine :+1: Just a little comment

**Review by chienandalu — APPROVED** [review]

**Pull request merged**